### PR TITLE
Add Create Flow screen

### PR DESCRIPTION
**Pull request by utkarsh-dixit**
- [x] Add a create flow screen
- [x] Add margin in navbar links and icons.

Light Mode:
Create Flow Screen Light

Dark Mode:
Create Flow Screen Dark

### CODE REVIEW

File: server/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+MONGODB=mongodb+srv://cheesetouched:68A2sS8P5XSv6Ur@hackathon.ea6nb.mongodb.net/onboardio?retryWrites=true&w=majority
```
Comment (flozender) [inline server/.env]: Oops

**Review by flozender — CHANGES_REQUESTED** [review]
LGTM, just see if you want to get rid of the .env

**Comment by flozender** [comment]
Also, does it work with dark mode as well? Asking since you used a third party multi select.

**Comment by utkarsh-dixit** [comment]
@flozender I'm still working on fixing the dark mode for react-select. 

Edit: Done now

**Review by pingrishabh — APPROVED** [review]
Wow. It's looking really good UI wise. 👍

**Comment by flozender** [comment]
Let's merge when you're ready! Closes #10

**Review by flozender — APPROVED** [review]

**Review by anje123 — APPROVED** [review]
This is looking good

**Pull request merged**